src/Redis/Initializer.php

```diff
@@ -128,7 +128,14 @@ public static function registerCommandsRediska(): void
 
     public static function getRediSearchVersion(Client $client): ?string
     {
-        $modules = $client->executeRaw('module', 'list') ?? [];
+        try {
+            $modules = $client->executeRaw('module', 'list') ?? [];
+        } catch (\Throwable $exception) {
+            if (strpos($exception->getMessage(), 'unknown command') === false) {
```

Comment:
Yoda style test is used (see PHP-CS-Fixer error: https://github.com/MacFJA/php-redisearch/runs/7054519075?check_suite_focus=true)
```suggestion
            if (false === strpos($exception->getMessage(), 'unknown command')) {
```